fe/fe-core/src/main/java/com/starrocks/server/MetadataMgr.java

```diff
@@ -243,7 +243,8 @@ public boolean createTable(CreateTableStmt stmt) throws DdlException {
                     ErrorReport.reportDdlException(ErrorCode.ERR_BAD_DB_ERROR, dbName);
                 }
 
-                if (listTableNames(catalogName, dbName).contains(tableName)) {
+                Table table = connectorMetadata.get().getTable(dbName, tableName);
```

Comment:
could we catch the exception to check if table exists?

Reply:
The exception has been caught in the `HiveMetadata#getTable` function. If caught, `null` will be returned.